client/service/src/client/wasm_override.rs

```diff
@@ -119,16 +119,13 @@ where
 	/// Scrapes a folder for WASM runtimes.
 	/// Returns a hashmap of the runtime version and wasm runtime code.
 	fn scrape_overrides(dir: &Path, executor: &E) -> Result<HashMap<u32, WasmBlob>> {
+
 		let handle_err = |e: std::io::Error | -> sp_blockchain::Error {
-			sp_blockchain::Error::Msg(format!("{}", e.to_string()))
+				sp_blockchain::Error::WasmOverrideIo(dir.to_owned(), e)
```

Comment:
seems like redundant indent level:
```suggestion
			sp_blockchain::Error::WasmOverrideIo(dir.to_owned(), e)
```